frontend/src/components/Application/Posts/views/PostEditor.vue

```diff
@@ -2,23 +2,71 @@
   <div class="container mt-2">
     <h1 class="mb-3">Post Something</h1>
     <v-form @submit.prevent="onSubmit">
-      <v-textarea
-        v-model="post_body"
-        class="form-control"
-        placehodler="What do you want to post?"
-        rows="3"
-      ></v-textarea>
+      <v-card>
+        <v-card-text>
+          <v-layout row wrap justify-space-around>
+            <v-flex xs8 sm9 text-xs-center>
```

Comment:
<v-flex> is deprecated in Vuetify 2.0, so this is going to have to be updated at some point.